eng/Signing.props

```diff
@@ -16,7 +16,6 @@
     <FileSignInfo Include="AzureSign.Core.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="Newtonsoft.Json.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="NuGetKeyVaultSignTool.Core.dll" CertificateName="3PartySHA2" />
-    <FileSignInfo Include="OpenVsixSignTool.Core.dll" CertificateName="3PartySHA2" />
```

Comment:
This dependency was removed in https://github.com/dotnet/sign/commit/f08287270bf7010efb07bd34d6c6e9732ef0a3dd#diff-5baf5f9e448ad54ab25a091adee0da05d4d228481c9200518fcb1b53a65d4156L27